src/components/MapEditor/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { memo , useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Map, Marker } from '@datapunt/react-maps';
+import { ViewerContainer } from '@datapunt/asc-ui';
+import { Zoom } from '@datapunt/amsterdam-react-maps/lib/components';
+import styled from '@datapunt/asc-core';
+import MAP_OPTIONS from 'shared/services/configuration/map-options';
```

Comment:
Importing the options here locks the component to those options where they should be configurable.

Reply:
moved it to the caller